src/unity/toolkits/supervised_learning/supervised_learning.cpp

```diff
@@ -350,6 +350,28 @@ std::vector<std::string>
   return tracking_metrics;
 }
 
+std::string supervised_learning_model_base::get_metric_display_name(const std::string& metric) const {
```

Comment:
Should this method be static?

Reply:
Yes, I think so. Originally I had in mind that some model types may want to override it; but the point of this change is to be consistent in how we are naming metrics, so we probably shouldn't allow it to be a per-model-type override.